src/tasks/task-angular-momentum-equality.cpp

```diff
@@ -96,12 +96,12 @@ namespace tsid
     }
     const Vector & TaskAMEquality::momentum_ref() const
     {
-      return m_ref.vel;
+      return m_ref.getValue();
     }
 
     const Vector & TaskAMEquality::dmomentum_ref() const
     {
-      return m_ref.acc;
+      return m_ref.getSecondDerivative();
```

Comment:
If the momentum is the value, then the momentum derivative should be the firstDerivative, shouldn't it?

Reply:
This was just a no-brain replace of `acc` to `getSecondDerivative()`. This `acc` comes from the initial commit: https://github.com/stack-of-tasks/tsid/commit/3cf00526fb92dc431456bf66cea3c00d7a1ee72d#diff-59c955f0c33d9ddd73e7bd4865a99dae5c8358352cd5cd3d73305172116977c3R103

I fixed it, without any change on the tests results, so I guess this was not tested :)